.gitignore

```diff
@@ -7,3 +7,8 @@
 /build
 /captures
 .externalNativeBuild
+*.apk
```

Comment:
https://www.gitignore.io/api/android%2Candroidstudio

Is what I used as reference